tf-modules/s3-credentials/README.md

```diff
@@ -0,0 +1,41 @@
+# Cumulus - S3 credentials endpoint module
+
+The Cumulus S3 credentials endpoint Terraform module deploys the S3
+credentials endpoint with a configuration targeted at Cumulus and NGAP.
```

Comment:
I think it would be nice to have a little bit more detail here. As a stand-alone module, it is less obvious what purpose this serves, so calling out the nature of the credential-dispensing and the additions of `/s3credentials` to the root of the provided API Gateway would be a value-add, IMO.

Reply:
Totally. I'll add that

Reply:
1f608a2ac8788a4747cad792c7e62261e93bd56c